pytorch_pfn_extras/_tensor.py

```diff
@@ -57,7 +57,9 @@ def as_ndarray(tensor: torch.Tensor) -> _NDArray:
         return tensor.detach().numpy()
     elif devtype == 'cuda':
         ensure_cupy()
-        return cupy.fromDlpack(torch.utils.dlpack.to_dlpack(tensor))
+        # TODO: Avoid using ``torch.utils.dlpack.to_dlpack``.
```

Comment:
* Please use `hasattr` to support CuPy v9 or earlier.
* Please add comment: `Blocked by PyTorch 1.10 bug (https://github.com/pytorch/pytorch/pull/67618)`